static/bidder-info/zeta_global_ssp.yaml

```diff
@@ -9,12 +12,14 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - audio
   site:
     mediaTypes:
       - banner
       - video
+      - audio
 userSync:
   redirect:
-    url: https://ssp.disqus.com/redirectuser?sid=GET_SID_FROM_ZETA&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&r={{.RedirectURL}}
+    url: https://ssp.disqus.com/redirectuser?sid={{.AccountID}}&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&r={{.RedirectURL}}
```

Comment:
`AccountID` is not a supported macro for user sync endpoints. The information can be optionally provided, so I'm not opposed to adding the macro - but I wouldn't expect it to always be present. How does your user sync behave when the  `AccountID` is unknown?

Reply:
@SyntaxNode In case of user sync, `sid` param is not critical for us and doesn't really impact the behavior. However, it would be great for us to know what publisher we do cookie sync with (for internal statistics and troubleshooting).

Reply:
@bsardo Thank you! I removed the macro from the redirect url for now. I'll open a separate PR to add support for it once I have time (hopefully, this year).